tests/container_test.py

```diff
@@ -89,5 +94,5 @@ def test_container_version_label_matches(version_container):
         exec(f.read(), pkg_vars)  # nosec
     project_version = pkg_vars["__version__"]
     assert (
-        version_container.labels["org.opencontainers.image.version"] == project_version
+        version_container.labels["version"] == project_version
```

Comment:
This should not be altered from the skeleton:
```suggestion
        version_container.labels["org.opencontainers.image.version"] == project_version
```

Reply:
I altered it because pytest kept failing that test. It would pass the test with the change.

It's changed back now but any idea how to resolve this?

<img width="763" alt="Screen Shot 2021-11-09 at 9 39 03 AM" src="https://user-images.githubusercontent.com/19600787/140944248-d7922271-bfd1-4bee-af31-2c39936c3a09.png">

Reply:
Thanks to @felddy who discovered to root cause of the problem here.  In your Dockerfile, you need to modify the two lines that contain:
```
cisagov/pca-report-library:0.0.1
```
so that they don't contain a version:
```
cisagov/pca-report-library
```

https://github.com/cisagov/pca-report-generator-docker/blob/4d038c2f7b9eab871cc0906b28694563547d5a7c/docker-compose.yml#L18-L26

By tagging your image with that `0.0.1` version, you are inadvertently breaking the build/test code, which operates on the `latest` version of your image.

Reply:
Resolved in 642ef1154b49013fc69e2514dcfe136c12782d29